blocks/library/latest-posts/index.js

```diff
@@ -97,17 +97,56 @@ registerBlockType( 'core/latest-posts', {
 
 		render() {
 			const { latestPosts } = this.state;
-			const { setAttributes } = this.props;
+			const { focus, setAttributes } = this.props;
+			const { displayPostDate, align, layout, columns } = this.props.attributes;
 
-			if ( ! latestPosts.length ) {
-				return (
-					<Placeholder
+			const inspectorControls = (
+				focus && (
+					<InspectorControls key="inspector">
+						<BlockDescription>
+							<p>{ __( 'Shows a list of your site\'s most recent posts.' ) }</p>
+						</BlockDescription>
+						<h3>{ __( 'Latest Posts Settings' ) }</h3>
+						<ToggleControl
+							label={ __( 'Display post date' ) }
+							checked={ displayPostDate }
+							onChange={ this.toggleDisplayPostDate }
+						/>
+						{ layout === 'grid' &&
+							<RangeControl
+								label={ __( 'Columns' ) }
+								value={ columns }
+								onChange={ ( value ) => setAttributes( { columns: value } ) }
+								min={ 2 }
+								max={ Math.min( MAX_POSTS_COLUMNS, latestPosts.length ) }
+							/>
+						}
+						<TextControl
+							label={ __( 'Number of posts to show' ) }
+							type="number"
+							min={ MIN_POSTS }
+							max={ MAX_POSTS }
+							value={ this.props.attributes.postsToShow }
+							onChange={ ( value ) => this.changePostsToShow( value ) }
+						/>
+					</InspectorControls>
+				)
+			);
+
+			const hasPosts = Array.isArray( latestPosts ) && latestPosts.length;
+			if ( ! hasPosts ) {
+				return [
+					inspectorControls,
+					<Placeholder key="placeholder"
 						icon="admin-post"
 						label={ __( 'Latest Posts' ) }
 					>
-						<Spinner />
-					</Placeholder>
-				);
+						{ ! Array.isArray( latestPosts ) ?
+							<Spinner /> :
+							__( 'No posts found.' )
+						}
```

Comment:
When using ternaries (or if/else structs, for that matter) we tend to prefer conditions that aren't negated. Also, specifically for ternaries (but not e.g. `&&`) we prefer to place the operators at the beginning of the breaking lines. So in this case:

```js
Array.isArray( latestPosts )
  ? __( 'No posts found.' )
  : <Spinner />
```

Reply:
I also favor these, but I wonder if a recent ESlint update forced the style applied here, in which case we should update the ESlint config.

Reply:
Oh, you're right, this has changed. @Soean, you can dismiss my comment on prepending the lines with the operators.

Once we update our ESLint config we can circle back, no worries.